content/kb/restoring-data-from-an-orphaned-replica-directory.md

```diff
@@ -0,0 +1,111 @@
+-----
+title: "Restoring Data from an Orphaned Replica Directory"
+authors:
+  - "Sushant Gaurav"
+draft: false
+date: 2025-09-02
+versions:
+- All
+categories:
+- "Data Recovery"
+- "Troubleshooting"
+-----
```

Comment:
CI failed because of header YAML.

```suggestion
---
title: "Restoring Data from an Orphaned Replica Directory"
authors:
- "Sushant Gaurav"
draft: false
date: 2025-09-02
versions:
- All
categories:
- "Data Recovery"
- "Troubleshooting"
---
```